File: pascal/control_flow/for_loop/for_loop_formatting.pas
program ForLoopFormatting;

var
  i: integer;

begin
  for i := 0 to 9 do
    writeln(i, '...');

  writeln();

  for i := 0 to 9 do
  begin
    writeln(i, '...');
  end;

  writeln();

  for i := 0 to 9 do writeln(i, '...');

  writeln();
end.

File: java/control_flow/for_loop/ForLoopFormatting.java
class ForLoopFormatting {
  public static void main(String[] args) {
    int i;

    for (i = 0; i < 10; i++)
      System.out.println(i + "...");

    System.out.println();

    for (int j = 0; j < 10; j++)
      System.out.println(j + "...");

    System.out.println();

    for (int j = 0; j < 10; j++) System.out.println(j + "...");

    System.out.println();

    for (int j = 0; j < 10; j++) {
      System.out.println(j + "...");
    }

    System.out.println();
  }
}
